sw/device/silicon_creator/mask_rom/e2e/command.h

```diff
@@ -0,0 +1,32 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_MASK_ROM_E2E_COMMAND_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_MASK_ROM_E2E_COMMAND_H_
+
+#include "sw/device/lib/ujson/ujson_derive.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+// clang-format off
+
+#define ENUM_TEST_COMMAND(_, value) \
+    value(_, ChipStartup)
```

Comment:
Is the idea that we would maintain a global command / struct list (below) across all tests? or would these be specific to a single test? (if the latter, then maybe we want to sub-folder these `commad.{c,h}` files?)

Reply:
I wasn't sure when I first wrote this, but after thinking about it, the answer is both:

`command.h` is an enumeration of all e2e test commands.  As such, it's Rust counterpart has been migrated into `opentitanlib::test_utils`.

Each individual test will have a certain set of structs it needs to exchange with its host-based test harness.  These should be declared in their own file and should become dependencies of the test code that needs them.